.github/workflows/Network_diff_upgrade.yml

```diff
@@ -0,0 +1,21 @@
+name: Network Diff Upgrade
+
+on:
+    workflow_dispatch:
+
+jobs:
+    diff_report:
+        name: Gamma Network Diff Report
+        uses: ./.github/workflows/Bytecode_diff_report.yml
+        secrets: inherit
+        with:
+            origin_environment: alpha
+            target_environment: gamma
+
+    upgrade_gamma_facets:
+        name: Upgrade Gamma Facets
+        needs: diff_report
+        uses: ./.github/workflows/Upgrade_network_facets.yml
```

Comment:
@jterzis the 2 workflows that this workflow refers to on lines 9 and 18 are showing as empty files. 

the other thing i'm noticing is that the upgrade_gamma_facets job has no way of reacting to the outcome of the first job, since these are not 2 steps on the same workflow but rather 2 separate workflows that get triggered in parallel. maybe that's intentional but calling it out just in case

Reply:
Where are you seeing that they are empty ? The two files are included actually in this pr so maybe that is why ? 

As for the second issue, these should be linked steps and I will make the change. I just need to merge the initial workflow file to be able to iterate on it in a subsequent branch. Otherwise I can't test the workflow in github.

Reply:
this is what im seeing, maybe the gui is tripping:

<img width="524" alt="image" src="https://github.com/user-attachments/assets/51d155be-0013-4bbf-b68a-a8b766bc910c">

Reply:
and about testing flow, sure sounds good to me. i'm approving 👍